src/components/common/PopupCall/components/Content.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { memo } from 'react'
+import callTo from '@/assets/images/callTo.svg'
+import callAway from '@/assets/images/callAway.svg'
+import avatarDefault from '@/assets/images/man.svg'
+
+interface Props {
+  type: 'call to' | 'call away'
+  name: string
+  avatar?: string
+  phoneNumber: string
+}
+
+const Content: React.FC<Props> = ({ type, name, avatar, phoneNumber }) => {
+  const call = type === 'call to' ? callTo : callAway
```

Comment:
Please explain this one

Reply:
cái này là xét xem type là gì. Nếu type là call to thì sẽ gán icon callTo cho biến call, sau đó em đưa biến call vào att src của thẻ img.